test/Microsoft.NET.TestFramework/Assertions/CommandResultExtensions.cs

```diff
@@ -7,9 +7,6 @@ namespace Microsoft.NET.TestFramework.Assertions
 {
     public static class CommandResultExtensions
     {
-        public static CommandResultAssertions Should(this CommandResult commandResult)
-        {
-            return new CommandResultAssertions(commandResult);
-        }
+        public static CommandResultAssertions Should(this CommandResult commandResult) => new CommandResultAssertions(commandResult);
```

Comment:
This file seems to have taken the Installer repo's history. Though, this change is fine for the file.